src/Facebook/FacebookBatchResponse.php

```diff
@@ -102,7 +102,7 @@ public function addResponse($key, $response)
 
         $httpResponseBody = isset($response['body']) ? $response['body'] : null;
         $httpResponseCode = isset($response['code']) ? $response['code'] : null;
-        $httpResponseHeaders = isset($response['headers']) ? $response['headers'] : [];
+        $httpResponseHeaders = isset($response['headers']) ? $this->normalizeBatchHeaders($response['headers']) : [];
```

Comment:
You and your efficient ways. :) Since [`array_column()` is a PHP 5.5 feature](http://php.net/array_column), we'll need to wait to add this until PHP SDK v6.0 (which I'm assuming will require PHP 5.5.) :)